<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Date</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="addButton">
      <property name="cursor">
       <cursorShape>UpArrowCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Добавить в файл дату на календаре</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="deleteButton">
      <property name="cursor">
       <cursorShape>UpArrowCursor</cursorShape>
      </property>
      <property name="toolTip">
       <string>Удалить из таблицы данные вида &quot;00.00.0000&quot;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 700 italic 54pt &quot;Ubuntu&quot;;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>Даты из файла</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 italic 14pt &quot;URW Bookman&quot;;
alternate-background-color: rgb(94, 92, 100);
background-color: rgb(61, 56, 70);
color: rgb(246, 245, 244);</string>
      </property>
      <property name="minimumDate">
       <date>
        <year>1752</year>
        <month>9</month>
        <day>14</day>
       </date>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
